### PR TITLE
done seekHelp midfi

### LINKED ISSUE #16: Finish Seek Help Page
## Todo: ##
- Now, our filter buttons should actually work. The button labels should reflect what's shown on Kevin's Figma. Clicking the buttons should also filter resources properly
- Make a new component under `/components` - you'll be creating a `resource` component that should take in props according to the data in supabase. Additionally, make sure that we have the dropdown functionality according to Kevin's figma.

image
image

## Useful Links: ##
https://reactnativeelements.com/docs/components/listitem_accordion

https://www.figma.com/design/KWJmW53rWnt8bE4U4iuc7B/JUST-DETENTION-INTERNATIONAL?node-id=6-6&node-type=canvas&t=2fvbdCmA0D8fCnd6-0

### PR DESCRIPTION
**Pull request by angelinetu**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- implemented dropdown resource card that displays a few lines of the summary and then the whole resource. 
- filter buttons now filter properly

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."
- will figure out how to make the resource card clickable to collapse it
- kevin said we're changing the filters and getting rid of the landing page

### Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @philipye314

### CODE REVIEW

File: src/components/Resource/Resource.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  return (
+    <View style={styles.resourceCard}>
+      <ListItem.Accordion
+        content={
+          <View style={styles.headerContent}>
+            <Text style={styles.orgName}>{org_name}</Text>
+            {!expanded && (
+              <Text style={styles.summaryPreview}>
+                {summary.slice(0, 60)}...
```
Comment (philipye314) [inline src/components/Resource/Resource.tsx]: ik u couldn't check during MPH but can you try to fix the caret on your iPad? It should be all the way on the right. Thanks!

File: src/components/Resource/Resource.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+interface SeekHelpProps {
+  org_name: string;
+  summary: string;
+  address: string;
+  phone_number: string;
+  hotline_number: string;
+  office_hours: string;
+  tags: string;
+}
```
Comment (philipye314) [inline src/components/Resource/Resource.tsx]: move to `/types`!

**Review by philipye314 — COMMENTED** [review]
looks mostly good, check the caret thing. Also lmk if u find a way to make the whole button collapsible/expandable

File: src/components/Resource/Resource.tsx (hunk 1 above)
Comment (angelinetu) [reply src/components/Resource/Resource.tsx]: i checked on my iPad and the caret is all the way on the right

Comment (angelinetu) [reply src/components/Resource/Resource.tsx]: wait jk i fixed it properly

**Review by philipye314 — COMMENTED** [review]
looks mainly good
TIME FOR STYLING!

**Review by philipye314 — APPROVED** [review]
LOOKIN GOOD BROTHER
